### ISSUE TITLE
[miopen] Link error

### ISSUE THREAD

**Issue by Laser-Cat** [rocm-arch/rocm-arch#56]
After fixing the boost issue as instructed by
https://github.com/ROCmSoftwarePlatform/MIOpen/issues/131
I get the same issue with
https://github.com/ROCmSoftwarePlatform/rocBLAS/issues/991,

i.e,
```
ld: ../lib/libMIOpen.so.1.0: undefined reference to `rocblas_gemm_ex'
ld: ../lib/libMIOpen.so.1.0: undefined reference to `rocblas_gemm_strided_batched_ex'
clang-10: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [driver/CMakeFiles/MIOpenDriver.dir/build.make:108: bin/MIOpenDriver] Error 1
make[2]: Leaving directory '/home/lasercat/rocm/rocm-arch/miopen/src/build'
make[1]: *** [CMakeFiles/Makefile2:4141: driver/CMakeFiles/MIOpenDriver.dir/all] Error 2
make[1]: Leaving directory '/home/lasercat/rocm/rocm-arch/miopen/src/build'
make: *** [Makefile:163: all] Error 2

```
Fix boost
[a.patch.txt](https://github.com/rocm-arch/rocm-arch/files/4333751/a.patch.txt)
corresponding PKGBUILD
[PKGBUILD.txt](https://github.com/rocm-arch/rocm-arch/files/4333752/PKGBUILD.txt)

**Comment by Laser-Cat** [comment]
I also tried to build rocm with tensile on as pointed by 
https://github.com/ROCmSoftwarePlatform/rocBLAS/issues/991,
and I got some python exceptions too:
[tensile_falied.log](https://github.com/rocm-arch/rocm-arch/files/4333758/tensile_falied.log)

**Comment by s-bernard** [comment]
This issue can be closed, fixed by #132.
